mongod/velopack.go

```
@@ -0,0 +1,39 @@
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-version"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+func (c *appRepository) RequiredIntermediateStep(ctx context.Context, owner, appName, channel, platform, arch, currentVersion, latestVersion string) (string, error) {
+	// CheckRequiredMigrationStep parses versions with version.Must
+	if _, err := version.NewVersion(currentVersion); err != nil {
+		return "", nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 12) but it returns nil

(nilerr)